packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -63,9 +85,12 @@ class FlutterGoldenFileComparator implements GoldenFileComparator {
   }) async {
     defaultComparator ??= goldenFileComparator;
 
-    // Prepare the goldens repo.
+    const Platform platform = LocalPlatform();
     goldens ??= GoldensClient();
-    await goldens.prepare();
+    if(platform.environment[_kServiceAccountKey] == null) {
```

Comment:
nit: space before after if (I believe you always want a space before a ( unless it is a function call.)